### PR TITLE
chore(website): add menu files for docs

### PR DESCRIPTION
**Pull request by Rhotimee**
#### Summary
Add menu files for docs

### CODE REVIEW

**Comment by melindafekete** [comment]
@nkuehn Would you happen to know how the top menu is configured for Custom Apps? We added the `top-menu.yaml` and `top-side-menu.yaml` files in `./[website/src/data/`, but now we have the old menu rendered alongside this new addition. Here is the **preview**. Thanks!

**Comment by nkuehn** [comment]
oh, this does not look good.  Sorry that I forgot that the default behavior is to add configuration data to the defaults provided by the kit instead of overriding them.  

The docs kit docs describe the config flag `overrideDefaultConfigurationData` including an example for this use case: https://commercetools-docs-kit.vercel.app/documentation/configuration/theme#available-options-for-the-theme  

> @nkuehn Would you happen to know how the top menu is configured for Custom Apps? We added the `top-menu.yaml` and `top-side-menu.yaml` files in `./[website/src/data/`, but now we have the old menu rendered alongside this new addition. Here is the **[preview](https://merchant-center-application-491pxn03h-commercetools.vercel.app/)**. Thanks!

File: website/src/data/top-side-menu.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- label: Sign up
+  href: /../getting-started/initial-setup
```
Comment (melindafekete) [inline website/src/data/top-side-menu.yaml:2]:
```suggestion
  href: https://docs.commercetools.com/getting-started/initial-setup
```

File: website/src/data/top-side-menu.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- label: Log in
+  href: /../docs/login
```
Comment (melindafekete) [inline website/src/data/top-side-menu.yaml:4]:
```suggestion
  href: https://docs.commercetools.com/docs/login
```

File: website/src/data/top-menu.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+- menu-title: Composable Commerce
+  items:
+    - label: Getting Started
+      href: /../getting-started/
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:4]:
```suggestion
      href: https://docs.commercetools.com/getting-started/
```

File: website/src/data/top-menu.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    - label: Merchant Center
+      href: /../merchant-center
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:6]:
```suggestion
      href: https://docs.commercetools.com/merchant-center
```

File: website/src/data/top-menu.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    - label: Tutorials
+      href: /../tutorials
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:8]:
```suggestion
      href: https://docs.commercetools.com/tutorials
```

File: website/src/data/top-menu.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    - label: HTTP API
+      href: /../api
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:10]:
```suggestion
      href: https://docs.commercetools.com/api
```

File: website/src/data/top-menu.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    - label: GraphQL API
+      href: /../api/graphql
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:12]:
```suggestion
      href: https://docs.commercetools.com/api/graphql
```

File: website/src/data/top-menu.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    - label: Import & Export
+      href: /../import-export
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:14]:
```suggestion
      href: https://docs.commercetools.com/import-export
```

File: website/src/data/top-menu.yaml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    - label: SDKs & Client Libraries
+      href: /../sdk
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:16]:
```suggestion
      href: https://docs.commercetools.com/sdk
```

File: website/src/data/top-menu.yaml [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    - label: Custom Applications
+      href: /../custom-applications
```
Comment (melindafekete) [inline website/src/data/top-menu.yaml:18]:
```suggestion
      href: https://docs.commercetools.com/custom-applications
```

**Review by melindafekete — APPROVED** [review]

File: website/src/data/top-side-menu.yaml (hunk 1 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-side-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-side-menu.yaml (hunk 2 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-side-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 3 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 4 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 5 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 6 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 7 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 8 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 9 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

File: website/src/data/top-menu.yaml (hunk 10 above)
Comment (CarlosCortizasCT) [reply website/src/data/top-menu.yaml]: Updated in this commit: 350add76898044cfb7056b5eb79892db0dda9cc1

**Review by melindafekete — APPROVED** [review]

**Review by CarlosCortizasCT — APPROVED** [review]
LGTM

**Pull request merged**